src/test/java/support/matchers/ResponseMatchers.java

```diff
@@ -13,17 +13,69 @@
 
 public class ResponseMatchers {
 
+  public static Matcher<Response> hasValidationErrorMessage(String expectedMessage) {
+    return new TypeSafeMatcher<>() {
+      @Override
+      protected boolean matchesSafely(Response response) {
+        if (response.getStatusCode() != 422) {
+          return false;
+        }
+
+        if (!response.getContentType().startsWith(ContentType.APPLICATION_JSON)) {
+          return false;
+        }
+
+          JsonArray errors = response.getJson().getJsonArray("errors");
+          if (errors != null && errors.size() == 1) {
+            JsonObject error = errors.getJsonObject(0);
+            JsonArray parameters = error.getJsonArray("parameters");
+
+            if (parameters != null && parameters.size() == 1) {
+              String message = error.getString("message");
+
+              return Objects.equals(expectedMessage, message);
+            }
+          }
+
+          return false;
+      }
+
+      @Override
+      public void describeTo(Description description) {
+        description
+          .appendText("Response has 422 status and 'message' - ")
+          .appendValue(expectedMessage);
+      }
+
+      @Override
+      protected void describeMismatchSafely(Response response,
+        Description mismatchDescription) {
+        mismatchDescription.appendText("Status: ").appendValue(
+            response.getStatusCode())
+          .appendText(", body: ");
```

Comment:
If I was going to split this statement over multiple lines, I would break it up on the method rather than parameters

(similarly for the other examples)

```suggestion
        mismatchDescription.appendText("Status: ")
          .appendValue( response.getStatusCode())
          .appendText(", body: ");
```

Reply:
@marcjohnson-kint  Good point! Changed.